.travis.yml

```diff
@@ -12,6 +12,16 @@ matrix:
         - BUILD_APIDOC=true
         # Auth token to push API documentation
         - secure: "JKeXk8p65hodb12PVRST6A90swsNubc+46EbSJGSghldIxbFWLBAlwU+KLeOMO4V0veu6k4lnMa50V0UYFZmoUsS6W0aL5Ybo98SpzXHiNLOmOluoqJoF9TBsOTCCRFbWbccgJyVEtulgRcdml96naS51lq9Sw/VO/N3Z472304="
+    - os: linix
+      language: python
+      python: 2.7.14
+      install:
+        - pip install pytest
+      script:
+        - pytest
+      before_script: echo ""
```

Comment:
Why do you want to suppress the global build steps anyway?

Reply:
Matrix `include` directive create separate build process. For that entry `before_script` is not required and it breaks script part, because it ends with `cd ...`